### PR TITLE
fixed PropTypes, changed Library to table, changed Borrow to only inc…

### PR DESCRIPTION
**Pull request by Drewbrad4**
…lude books not owned by user

### CODE REVIEW

File: app/javascript/components/Pages/Borrow.js [hunk 1]
```diff
@@ -4,44 +4,45 @@ import { Link } from 'react-router-dom';
...
- }
\ No newline at end of file
+Borrow.propTypes = {
+    books: PropTypes.arrayOf(PropTypes.shape({
+        title: PropTypes.string,
+        author: PropTypes.string,
+        description: PropTypes.string,
+        pages: PropTypes.integer,
+        series: PropTypes.string,
+        pick_up_date: PropTypes.instanceOf(Date),
```
Comment (sturdynut) [inline app/javascript/components/Pages/Borrow.js:45]: I think we want to update these as well to be `PropTypes.string`?

**Review by sturdynut — CHANGES_REQUESTED** [review]
Looks great, I think we missed a prop type update.

File: app/javascript/components/Pages/Borrow.js (hunk 1 above)
Comment (Drewbrad4) [reply app/javascript/components/Pages/Borrow.js]: Yes, just updated. Thank you!

**Comment by devJoelMichael** [comment]
Looks good

**Pull request merged**